### DISCUSSION TITLE
Installation appears stuck.. until corepack is enabled

### DISCUSSION THREAD

**Discussion by Klrfl** [medusajs/medusa#10770] [Q&A]
This wasted a lot of time...

So I was trying Medusa for the very first time, but the installation with `npx create-medusa-app@latest` just isn't working. It will show the loader, stay there for a few minutes and doesn't do anything. I decided to use `pnpm dlx` instead of `npx` and it worked, until the dependency installation step.

From there I went here and [read this](https://github.com/medusajs/medusa/discussions/6010), and decided to add the flag `--verbose` to the installation command (the one with `pnpm dlx`). And turns out Medusa needs Yarn for installation? I turned on corepack, and lo and behold, the installation prompted me to install Yarn and it succeeded.

From my understanding, Medusa cannot use any other package manager other than Yarn, does it? If this is true, you should disclose this in your documentation so as to not waste so much of people's time like I did...

**Comment by maclive** [comment]
me too

**Comment by adevinwild** [comment]
I know that yarn is recommended, but it's mainly because of corepack that I've been using [Bun](https://bun.sh/) on Medusa projects since a few months and no issue at all, even on V1 projects at a production level 👍🏼

**Comment by maclive** [comment]
not works same thing hang
